Dockerfile

```diff
@@ -1,147 +1,60 @@
-ARG unitycatalog_uid=185
-ARG unitycatalog_home="/opt/unitycatalog"
-ARG unitycatalog_repo="${unitycatalog_home}/repo"
-ARG unitycatalog_jar="server/target"
-ARG unitycatalog_jars="${unitycatalog_home}/jars"
-ARG unitycatalog_etc="etc"
-ARG unitycatalog_bin="bin"
-ARG unitycatalog_user_name="unitycatalog"
-ARG unitycatalog_user_home="home"
-ARG unitycatalog_user_basedir="${unitycatalog_home}/${unitycatalog_user_home}"
-# Specify any custom parameters necessary to generate
-# the Uber-Jar by SBT. 
-# Note: The default allocated heam memory size is too small
-# and will cause the process to fail when attempting to compile
-# and generate the Uber-Jar. Therefore it is important to choose
-# a size large enough for the compiler to run. 
-ARG sbt_args="-J-Xmx2G"
-# FIXME Pass it from the outside
-ARG unitycatalog_version="0.2.0-SNAPSHOT"
-ARG jars_directory="server/target/jars"
+# syntax=docker.io/docker/dockerfile:1.7-labs
+ARG HOME="/unitycatalog"
```

Comment:
Why not stick to `/opt/unitycatalog`. It's shorter yet we don't really care, do we? `/opt` looks as the right place to me.